app/main.py

```diff
@@ -1,8 +1,25 @@
+# main.py content
+
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for pers in people:
+        Person(pers["name"], pers["age"])
+
+    for pers in people:
+        person = Person.people[pers["name"]]
+        if "wife" in pers and pers["wife"]:
+            wife_name = pers["wife"]
+            wife = Person.people[wife_name]
+            setattr(person, "wife", wife)
+            setattr(wife, "husband", person)
+
+    return list(Person.people.values())
```

Comment:
You can optimize this so you create a Person, set wife/husband attributes and form a list that you will return in the end in one iteration